conda_build/utils.py

```diff
@@ -985,9 +984,10 @@ def sys_path_prepended(prefix):
 
 
 @contextlib.contextmanager
-def path_prepended(prefix):
+def path_prepended(prefix, prepend_prefix=True):
+    # FIXME: Unclear why prepend_prefix=True for all platforms.
     old_path = os.environ['PATH']
-    os.environ['PATH'] = prepend_bin_path(os.environ.copy(), prefix, True)['PATH']
+    os.environ['PATH'] = prepend_bin_path(os.environ.copy(), prefix, prepend_prefix)['PATH']
```

Comment:
No idea why this is `prepend_prefix=True` for all platforms.
For now, I left it that way and only changed it to `False` in `build.write_build_scripts` where I needed it.
I did not look at the other places where `path_prepended` is used. For me, this is also out of scope for this PR, but I'd like us to fix it, hence the added `FIXME`-note.